packages/oracle/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "bugs": "https://github.com/sequelize/sequelize/issues",
+  "description": "Oracle Database Connector for Sequelize",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./lib/index.d.mts",
+        "default": "./lib/index.mjs"
+      },
+      "require": {
+        "types": "./lib/index.d.ts",
+        "default": "./lib/index.js"
+      }
+    }
+  },
+  "files": [
+    "lib"
+  ],
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "sideEffects": false,
+  "homepage": "https://sequelize.org",
```

Comment:
Could point to https://sequelize.org/docs/v7/databases/oracle/ (assuming that's the URL where the oracle documentation will be located)

Reply:
updated.